### ISSUE TITLE
New setting to disable image downloading

### ISSUE THREAD

**Issue by bsclifton** [brave/brave-browser#8370]
## Description 
Users should be able to turn off image downloading. This would be extremely useful for folks using a screenreader.

Feature request from user via DM:
> The setting is missing to avoid downloading and rendering images: if that's present, already fast speeds will be blazing fast for screen reader users

**Comment by bsclifton** [comment]
cc: @rebron for prioritization

**Comment by W4RH4WK** [comment]
If I understand this topic correctly, this is now available on desktop.
![image](https://user-images.githubusercontent.com/328798/105985016-34628000-609b-11eb-8715-d5122718d651.png)

However, I am missing this feature on Android. Can somebody explain why this feature is absent on Android despite being available on desktop?

**Comment by facuescobar** [comment]
> Closing as not actionable. Picked up as part of chromium update.
> 
> brave://settings/content/images
> 
> <img alt="Screen Shot 2021-10-06 at 9 34 50 AM" width="678" src="https://user-images.githubusercontent.com/5951041/136246160-ed6f82be-725f-4f58-9a55-dc6ad2e07674.png">

Just a note regarding this feature:

This setting is not easy to find. 

One cool thing that Chrome had about this feature is that if I would type "Images" in the search input, the option would show up.

For Brave, the option doesn't show up, so you have to search for Site Shields and Settings to access to it.

I think it would be great if I can find it easily by typing the option name as you can do it in Chrome. 

Anyway, thanks for sharing it!
